### PR TITLE
[DOC] Improve documentation configuration

### LINKED ISSUE #117: Example render for 0.4 stable website
The examples did not render on the stable version of the doc: https://mne.tools/mne-icalabel/stable/auto_examples/index.html
It would be good to checkout the 0.4.0 version, rebuild the doc locally, and upload it on the gh-pages branch in the correct directory.

### PR DESCRIPTION
**Pull request by mscheltienne**
Closes #117 
Let's see if that still builds 😄

### CODE REVIEW

**Comment by mscheltienne** [comment]
You can have a look at the render doc here: https://output.circle-artifacts.com/output/job/931e20d3-b8a6-4fde-8073-889610145d90/artifacts/0/stable/index.html

The `Check the rendered docs here!` link does not work.

**Comment by mscheltienne** [comment]
Updated link: https://output.circle-artifacts.com/output/job/414563d7-1fb9-4dd6-972a-e7a9d3367718/artifacts/0/stable/index.html

File: doc/index.rst [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-that build upon the ICLabel model.
+``mne-icalabel`` is a Python package for labeling independent components that
+stem from an `Independent Component Analysis (ICA) <ica wikipedia_>`_.
+
+Scalp electroencephalography (EEG) and magnetoencephalography (MEG) analysis is
+typically very noisy and contains various non-neural signals, such as heartbeat
+artifacts. `Independent Component Analysis (ICA) <ica wikipedia_>`_ is a common
+procedure to remove these artifacts. However, removing artifacts requires
+manual annotation of ICA components, which is subject to human error and very
+laborious when operating on large datasets. The first few version of
```
Comment (adam2392) [inline doc/index.rst:14]:
```suggestion
laborious when operating on large datasets. The first few versions of
```

**Review by adam2392 — APPROVED** [review]
LGTM!

**Pull request merged**